src/components/Person/Person.jsx

```diff
@@ -1 +1,13 @@
-// export const Person = ({ person }) => ();
+import './Person.scss';
+
+export const Person = ({ person }) => (
+  <section className="Person">
+    <h2 className="Person__name">My name is {person.name}</h2>
+    {person.age && <p className="Person__age">I am 37</p>}
```

Comment:
Fix it like this after destruction
```suggestion
    {age && <p className="Person__age">{`I am ${age}`}</p>}
```

Reply:
Hi, I didn't notice the destructuring, but the tests themselves required me to write 37 for age

Reply:
Hi, 37 is an age value from Misha object
![image](https://github.com/mate-academy/react_person/assets/80700247/c56e21cd-1d99-4103-bda1-c229293106f6)